File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciorder/d/HpBeyondEnum.java
package iih.ci.ord.ciorder.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface HpBeyondEnum {

    @DmEnumDesc(name="保内诊断",description="保内诊断")
    public static final String HPDIAG="0"; //保内诊断
    @DmEnumDesc(name="保外诊断",description="保外诊断")
    public static final String HPEXTERNALDIAG="1"; //保外诊断
    @DmEnumDesc(name="非医保就诊",description="非医保就诊")
    public static final String NONMEDICARE="9"; //非医保就诊
}	

File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciorder/d/HpBeyondEnum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.ciorder.d
{
	public class HpBeyondEnum {
        [Description("保内诊断")]
		public const String HPDIAG="0";//保内诊断
        [Description("保外诊断")]
		public const String HPEXTERNALDIAG="1";//保外诊断
        [Description("非医保就诊")]
		public const String NONMEDICARE="9";//非医保就诊
   }
}
